app/src/main/res/layout/image_region_selection_interaction_item.xml

```diff
@@ -43,20 +36,38 @@
     app:questionViewMarginStart="@{@dimen/interaction_item_question_view_margin_start}"
     app:questionViewMarginTop="@{@dimen/interaction_item_question_view_margin_top}">
 
-    <org.oppia.android.app.player.state.ImageRegionSelectionInteractionView
-      android:id="@+id/image_click_interaction_image_view"
-      android:layout_width="wrap_content"
+    <FrameLayout
+      android:id="@+id/interaction_container_frame_layout"
+      android:layout_width="match_parent"
       android:layout_height="wrap_content"
-      android:adjustViewBounds="true"
-      app:clickableAreas="@{viewModel.selectableRegions}"
-      app:entityId="@{viewModel.entityId}"
-      app:imageUrl="@{viewModel.imagePath}"
-      app:onRegionClicked="@{(region) -> viewModel.onClickableAreaTouched(region)}"
-      app:overlayView="@{interactionContainerFrameLayout}" />
+      android:background="@drawable/drag_drop_white_background"
+      android:descendantFocusability="beforeDescendants"
+      android:focusableInTouchMode="true"
+      android:layoutDirection="ltr"
```

Comment:
This & the ltr below seems like the main fix. Why not fix this within the custom view, instead? It seems preferable to have custom views properly support RTL rather than disabling it at the parent.

Reply:
Imageview is not supporting layout direction. So I fixed it for the parent layout..

Reply:
Looks like a solution was found at the view level & this is now resolved.